storm-core/src/jvm/org/apache/storm/utils/Time.java

```diff
@@ -98,10 +104,25 @@ public static long currentTimeMillis() {
             return System.currentTimeMillis();
         }
     }
-    
+
+    public static long toMillis (int secs) {
+        return 1000*(long) secs;
+    }
+    public static long toMillis (String secs) {
+        return 1000*Long.parseLong(secs);
+    }
+
     public static int currentTimeSecs() {
         return (int) (currentTimeMillis() / 1000);
     }
+
+    public static int delta(int timeInSeconds) {
```

Comment:
Can we rename this to deltaSecs, because it is not consistent if times here are seconds or milliseconds.